### ISSUE TITLE
says that manifest file isnt readablle or isnt there.

### ISSUE THREAD

**Issue by minikdrift** [SintcoLTD/sparxmaths#28]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by NajmAjmal** [comment]
Hi, @minikdrift

I believe you have made a small mistake when attempting to load the extension. You have simply selected the zip without extracting it, or you have selected the main repository folder, instead of the extension sub-folder.

When downloading the extension, make sure to follow these steps.

1. When downloaded, extract all the contents from the file
2. When adding the extension, Do not just select the fold that has been extracted. 
3. Look through the sub-folders until you see the folder called “extension”
4. Select the “extension” folder.

To put it simply. When adding the extension, select the extension folder after unzipping the program.

This should work if not re-open this issue.


Thanks, Najm
